addons/payment_alipay/tests/test_alipay.py

```diff
@@ -99,13 +99,14 @@ def test_03_redirect_form_with_fees(self):
             'fees_dom_var': 0.35,
             'fees_int_fixed': 1.5,
             'fees_int_var': 0.50,
+            'main_currency_id': self.currency_euro,
         })
-
+        self.currency = self.currency_euro
         transaction_fees = self.currency.round(
             self.alipay._compute_fees(
                 self.amount,
                 self.currency,
-                self.partner.country_id,
+                self.env.ref('base.be'),
```

Comment:
You fixed some problem of conversion between different currencies, why, then, changing the setup to only use euro, as I guessed that's why you had to change this?

Reply:
done